tools/vscode-azurewebpubsub/src/workflows/common/contexts.ts

```diff
@@ -43,3 +44,21 @@ export interface IUpdateServiceContext extends IPickServiceContext {
 export interface ICreateOrUpdateHubSettingContext extends IPickHubSettingContext {
     hubProperties?: WebPubSubHubProperties;
 }
+
+export type MetricName = string;
+export enum KnownMetricNameEnum {
+    ServerLoad = "ServerLoad",
+    InboundTraffic = "InboundTraffic",
+    OutboundTraffic = "OutboundTraffic",
+    ConnectionQuotaUtilization = "ConnectionQuotaUtilization",
+    ConnectionCount = "ConnectionCount",
```

Comment:
Actually, wps is using the internal metrics name "TotalConnectionCount" for the connection count. Please double check
`metricnames=TotalConnectionCount&aggregation=maximum&metricNamespace=microsoft.signalrservice%2Fwebpubsub&validatedimensions=false&api-version=2019-07-01`

Reply:
corrected it to "TotalConnectionCount". confirmed it works